mobile/lib/extensions/build_context.dart

```diff
@@ -1,8 +1,10 @@
 import 'package:flutter/material.dart';
 import 'package:radioaktywne/resources/colors.dart';
+import '../resources/text_styles.dart';
```

Comment:
avoid relative paths in imports in case of any file location changes

Reply:
fixed in the latest commit